adcm_aio_client/core/objects/_base.py

```diff
@@ -48,7 +49,8 @@ def _construct_child[Child: "InteractiveChildObject"](
     def _clear_cache(self: Self) -> None:
         for name in self._delete_on_refresh:
             # works for cached_property
-            delattr(self, name)
+            with suppress(AttributeError):
```

Comment:
please leave a comment for suppress

as I understand it's for cases when value isn't cached, but it's a thing easy to forget

Reply:
done